nni/algorithms/compression/v2/pytorch/pruning/iterative_pruner.py

```diff
@@ -227,6 +224,11 @@ class SimulatedAnnealingPruner(IterativePruner):
         Cool down rate of the temperature.
     perturbation_magnitude : float
         Initial perturbation magnitude to the sparsities. The magnitude decreases with current temperature.
+    pruning_algorithm : str
+        Supported pruning algorithm ['level', 'l1', 'l2', 'fpgm', 'slim', 'apoz', 'mean_activation', 'taylorfo', 'admm'].
+        This iterative pruner will use the chosen corresponding pruner to prune the model in each iteration.
```

Comment:
chosen corresponding pruner -> chosen one-shot pruner

Reply:
we don't explain what is one-shot pruner in v2